src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let tempSum = 0;
+  let sum;
```

Comment:
you can set `let sum = 0` and inside the function you can set `
let tempSum  =  sum
sum = 0
return tempSum
`
with this you can have only one global variable.